code/modules/organs/body_modifications.dm

```diff
@@ -66,7 +66,9 @@ var/global/list/modifications_types = list(
 	if(department_specific.len)
 		if(H && H.mind)
 			var/department = H.mind.assigned_job.department
-			if(!department || !department_specific.Find(department))
+			if(!department)
+				return FALSE
+			else if(!department_specific.Find(department))
```

Comment:
```suggestion
			if(!department_specific.Find(department))
```

Reply:
so it turns out it doesnt work without the else